packages/sqs/.eslintrc.json

```diff
@@ -23,7 +23,7 @@
     "@typescript-eslint/no-empty-interface": "warn",
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/no-use-before-define": "off",
-    "@typescript-eslint/no-non-null-assertion": "warn",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
Looks like we have the same here, maybe I am missing something and this is needed so please let me know if it is the case 🙇

Reply:
will address in a follow-up PR